src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class declaration.